virt/virt-4-11-release-notes.adoc

```diff
@@ -42,6 +42,7 @@ The SVVP Certification applies to:
 ** Intel and AMD CPUs.
 
 //CNV-16067 CNV revalidated on a 3-node compact cluster
+* You can now deploy {VirtProductName} on a xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installation-three-node-cluster_installing-bare-metal[three-node cluster] with no workers.
```

Comment:
Nit: I know that we have used "workers" in the CNV docs, but the link uses "compute" nodes/machines. For consistency, I'd suggest switching "with no workers" to something like "with zero compute nodes." 

I did a little digging in the team-docs channel, and it seems that "compute" is preferred because it's more of an OCP-specific term, reflecting that they do more than vanilla K8s